spec/tags/truffle/launcher_tags.txt

```diff
@@ -43,14 +43,6 @@ aot:The launcher '-J:cp ' adds the jar
 aot:The launcher '--jvm.classpath=' adds the jar
 aot:The launcher '--jvm.cp=' adds the jar
 slow:The launcher enables deterministic hashing if -Xhashing.deterministic is set
-aot:The launcher 'gem' in `RbConfig::CONFIG['bindir']` directory runs when symlinked
```

Comment:
Were these just old? I'm failing to see how the changes you made would affect anything having to do with following symlinks.

Reply:
No, it's solving this. Before, the image would be output in `svmbuild/ruby` and the other launchers would never find the image, or vice-versa.